includes/functions/llms.functions.templates.dashboard.php

```diff
@@ -288,6 +288,80 @@ function lifterlms_template_my_courses_loop( $student = null, $preview = false )
 	}
 }
 
+if ( ! function_exists( 'lifterlms_template_my_favorites_loop' ) ) {
```

Comment:
https://github.com/gocodebox/lifterlms/blob/trunk/docs/coding-standards.md#functions--variables
`llms_` is the correct prefix.

Reply:
fixed. will remember next time.